Oracle/OracleJava7JDK.munki.recipe

```diff
@@ -5,43 +5,146 @@
     <key>Description</key>
     <string>Downloads latest release version of Oracle Java 7 JDK and imports into a Munki repository.</string>
     <key>Identifier</key>
-    <string>com.github.jaharmi.munki.OracleJava7JDK</string>
+    <string>com.github.jaharmi.munki.OracleJava7JDK.munki</string>
```

Comment:
I don't believe this should add `.munki` to the end of this Identifier. That will break override recipes.